slack_sdk/web/async_slack_response.py

```diff
@@ -203,5 +203,5 @@ def validate(self):
             and (isinstance(self.data, bytes) or self.data.get("ok", False))
         ):
             return self
-        msg = "The request to the Slack API failed."
+        msg = f"The request to the Slack API failed. (url: {self.api_url})"
```

Comment:
While running tests, I found that this error message does not have sufficient information.